pkg/katatestutils/constraints.go

```diff
@@ -0,0 +1,355 @@
+// Copyright (c) 2019 Intel Corporation
```

Comment:
Should there be some accompanying documentation with these additions - detailing what/how/where you can apply constraints to the test files?